9999999999{"name": "pyodbc", "version": "4.0.39", "summary": "DB API Module for ODBC", "requires_dist": null, "requires_python": ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*", "files": [{"file": "pyodbc-4.0.39-cp310-cp310-macosx_10_9_x86_64.whl", "hash": "sha256:74af348dbaee4885998858daf50c8964e767629ecf6c195868b016367b0bb861"}, {"file": "pyodbc-4.0.39-cp310-cp310-macosx_11_0_arm64.whl", "hash": "sha256:0f5901b57eaef0761f4cf02bca8e7c63f589fd0fd723a79f6ccf1ea1275372e5"}, {"file": "pyodbc-4.0.39-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:e0db69478d00fcd8d0b9bdde8aca0b0eada341fd6ed8c2da84b594b928c84106"}, {"file": "pyodbc-4.0.39-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:c5faf2870e9d434c6a85c6adc1cdff55c0e376273baf480f06d9848025405688"}, {"file": "pyodbc-4.0.39-cp310-cp310-win32.whl", "hash": "sha256:62bb6d7d0d25dc75d1445e539f946461c9c5a3643ae14676b240f71794ea004f"}, {"file": "pyodbc-4.0.39-cp310-cp310-win_amd64.whl", "hash": "sha256:8eb5547282dc73a7784ce7b99584f68687dd85543538ca6f70cffaa6310676e7"}, {"file": "pyodbc-4.0.39-cp311-cp311-macosx_10_9_x86_64.whl", "hash": "sha256:530c1ac37ead782803b44fb1934ba4c68ed4a6969f7475cb8bc04ae1da14486e"}, {"file": "pyodbc-4.0.39-cp311-cp311-macosx_11_0_arm64.whl", "hash": "sha256:1f7fb65191926308f09ce75ae7ccecf89310232ee50cdea74edf17ee04a9b068"}, {"file": "pyodbc-4.0.39-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:5ec009180fcd7c8197f45d083e6670623d8dfe198a457ca2a50ebb1bafe4107f"}, {"file": "pyodbc-4.0.39-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:400e911d54980098c6badadecc82385fc0d6a9057db525d63d2652317df43efe"}, {"file": "pyodbc-4.0.39-cp311-cp311-win32.whl", "hash": "sha256:f792677b88e1dde12dab46de8647620fc8171742c02780d51744f7b1b2135dbc"}, {"file": "pyodbc-4.0.39-cp311-cp311-win_amd64.whl", "hash": "sha256:3d9d70e1635d35ba3aee3df216ec8e35f2824909f43331c0112b17f460a93923"}, {"file": "pyodbc-4.0.39-cp36-cp36m-macosx_10_9_x86_64.whl", "hash": "sha256:c1a59096f1784d0cda3d0b8f393849f05515c46a10016edb6da1b1960d039800"}, {"file": "pyodbc-4.0.39-cp36-cp36m-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:1b3467157661615d5c30893efa1069b55c9ffa434097fc3ae3739e740d83d2ec"}, {"file": "pyodbc-4.0.39-cp36-cp36m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:af027a60e84274ea08fad1c75991d37a5f1f6e8bcd30f6bda20db99f0cdfbc7d"}, {"file": "pyodbc-4.0.39-cp36-cp36m-win32.whl", "hash": "sha256:64c1de1263281de7b5ce585b0352746ab1a483453017a8589f838a79cbe3d6d9"}, {"file": "pyodbc-4.0.39-cp36-cp36m-win_amd64.whl", "hash": "sha256:27d1b3c3159673b44c97c878f9d8056901d45f747ce2e0b4d5d99f0fb6949dc7"}, {"file": "pyodbc-4.0.39-cp37-cp37m-macosx_10_9_x86_64.whl", "hash": "sha256:efccc11dff6fba684a74ae1030c92ff8b82429d7f00e0a50aa2ac6f56621cd9f"}, {"file": "pyodbc-4.0.39-cp37-cp37m-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:ea08e9379c08663d7260e2b8a6c451f56d36c17291af735191089f8e29ad9578"}, {"file": "pyodbc-4.0.39-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:b36fe804d367d01ad81077fa524a36e667aabc3945e32564c7ef9595b28edfa9"}, {"file": "pyodbc-4.0.39-cp37-cp37m-win32.whl", "hash": "sha256:72d364e52f6ca2417881a23834b3a36733c09e0dcd4760f49a6b864218d98d92"}, {"file": "pyodbc-4.0.39-cp37-cp37m-win_amd64.whl", "hash": "sha256:39f6c56022c764309aa7552c0eb2c58fbb5902ab5d2010d42b021c0b205aa609"}, {"file": "pyodbc-4.0.39-cp38-cp38-macosx_10_9_x86_64.whl", "hash": "sha256:ebcb900fcaf19ca2bc38632218c5d48c666fcc19fe38b08cde001917f4581456"}, {"file": "pyodbc-4.0.39-cp38-cp38-macosx_11_0_arm64.whl", "hash": "sha256:a3e133621ac2dad22d0870a8521c7e82d4270e24ce02451d64e7eb6a40ad0941"}, {"file": "pyodbc-4.0.39-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:05a0912e852ebddaffa8f235b0f3974475021dd8eb604eb46ea67af06efe1239"}, {"file": "pyodbc-4.0.39-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:6353044b99c763aeec7ca1760b4340298504d8ee544fdcab3c380a2abec15b78"}, {"file": "pyodbc-4.0.39-cp38-cp38-win32.whl", "hash": "sha256:a591a1cf3c251a9c7c1642cfb3774119bf3512f3be56151247238f8a7b22b336"}, {"file": "pyodbc-4.0.39-cp38-cp38-win_amd64.whl", "hash": "sha256:8553eaef9f8ec333bbddff6eadf0d322dda34b37f4bab19f0658eb532037840c"}, {"file": "pyodbc-4.0.39-cp39-cp39-macosx_10_9_x86_64.whl", "hash": "sha256:9253e746c5c94bf61e3e9adb08fb7688d413cb68c06ebb287ec233387534760a"}, {"file": "pyodbc-4.0.39-cp39-cp39-macosx_11_0_arm64.whl", "hash": "sha256:a6f4067f46aaa78e77e8a15ade81eb21fb344563d245fb2d9a0aaa553c367cbd"}, {"file": "pyodbc-4.0.39-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", "hash": "sha256:cdf5a27e6587d1762f7f0e35d6f0309f09019bf3e19ca9177a4b765121f3f106"}, {"file": "pyodbc-4.0.39-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", "hash": "sha256:fe4ee87b88867867f582dd0c1236cd982508db359a6cbb5e91623ceb6c83e60a"}, {"file": "pyodbc-4.0.39-cp39-cp39-win32.whl", "hash": "sha256:42649ed57d09c04aa197bdd4fe0aa9ca319790b7aa86d0b0784cc70e78c426e5"}, {"file": "pyodbc-4.0.39-cp39-cp39-win_amd64.whl", "hash": "sha256:305c7d6337e2d4c8350677cc641b343fc0197b7b9bc167815c66b64545c67a53"}, {"file": "pyodbc-4.0.39.tar.gz", "hash": "sha256:e528bb70dd6d6299ee429868925df0866e3e919c772b9eff79c8e17920d8f116"}], "yanked": false, "_cache_version": "2.0.0"}